subsys/app_event_manager/app_event_manager_priv.h

```diff
@@ -82,34 +82,37 @@ extern "C" {
 #define _EM_TAG_NAME(prefix) _CONCAT(prefix, _tag)
 
 /* Zero-length subscriber to be used as a tag. */
-#define _EVENT_SUBSCRIBERS_TAG(ename, marker)							\
-	const struct {} _EM_TAG_NAME(_EVENT_SUBSCRIBERS_SECTION_PREFIX(ename, marker))		\
+#define _APP_EVENT_SUBSCRIBERS_TAG(ename, marker)					\
```

Comment:
Also here: APP prefix added before internal API. Do we want to add it here?

Reply:
Also in other places related to the internal API

Reply:
Changed all macros to use _APP_XYZ format